docs/source/explanation/algorithms/segmentation/semantic_segmentation.rst

```diff
@@ -25,7 +25,58 @@ For the supervised training we use the following algorithms components:
 
 - ``Loss function``: We use standart `Cross Entropy Loss <https://en.wikipedia.org/wiki/Cross_entropy>`_  to train a model.
 
-- Additionally, we use the **early stopping** to add adaptability to the training pipeline and prevent overfitting.
+- ``Training technique``
+    - ``Early stopping``: To add adaptability to the training pipeline and prevent overfitting. You can use early stopping like the below command.
+
+      .. code-block::
+
+        $ otx train {TEMPLATE} ... \
+            params \
+            --learning_parameters.enable_early_stopping=True \      # is early stopping used
+            --learning_parameters.early_stop_start=3 \              # the number of epochs (iters) in which early stopping proceeds
```

Comment:
Please, use only ``--learning_parameters.enable_early_stopping=True``  for the example. Also, the early stopping is on by default. Maybe it is even redundant and misleading information. In the tutorials we have some examples how to modify parameters. I suggest deleting it at all

Reply:
Done